scout/commands/load/case.py

```diff
@@ -115,7 +115,18 @@ def case(
 
     try:
         adapter.load_case(config_data, update, keep_actions)
+    except SyntaxError as err:
+        LOG.error(
+            "SyntaxError {} missing when loading '{}' {}".format(
+                err, config.name, traceback.format_exc()
+            )
+        )
+        raise click.Abort()
+    except KeyError as err:
+        LOG.error(
+            "KeyError {} when loading '{}' {}".format(err, config.name, traceback.format_exc())
+        )
+        raise click.Abort()
     except Exception as err:
-        LOG.error("Something went wrong during loading")
-        LOG.warning(err)
+        LOG.error("Unhandled Exception: {}".format(traceback.format_exc()))
```

Comment:
Maybe print the error here as well, or was there good reason to hide it?

Reply:
Printing `err` doesn't add information when the stack trace is printed. This code...

```
        LOG.error("Unhandled Exception: {}".format(traceback.format_exc()))
        LOG.error("err: {}".format(err))
```

Will produce...

```
2021-06-08 13:01:48 mikaellmbp scout.commands.load.case[76637] ERROR Unhandled Exception: Traceback (most recent call last):
  File "/miniconda3/envs/py3.7/lib/python3.7/site-packages/scout_browser-4.36-py3.7.egg/scout/commands/load/case.py", line 117, in case
    adapter.load_case(config_data, update, keep_actions)
  File "/miniconda3/envs/py3.7/lib/python3.7/site-packages/scout_browser-4.36-py3.7.egg/scout/adapter/mongo/case.py", line 644, in load_case
    rank_threshold=case_obj.get("rank_score_threshold", 5),
  File "/miniconda3/envs/py3.7/lib/python3.7/site-packages/scout_browser-4.36-py3.7.egg/scout/adapter/mongo/variant_loader.py", line 637, in load_variants
    vcf_obj = VCF(variant_file)
  File "cyvcf2/cyvcf2.pyx", line 185, in cyvcf2.cyvcf2.VCF.__init__
Exception: bad path: scout/demo/643594.clinical.vcf.gz.bad_path

2021-06-08 13:01:48 mikaellmbp scout.commands.load.case[76637] ERROR err: bad path: scout/demo/643594.clinical.vcf.gz.bad_path
```